torch/csrc/dynamo/eval_frame.c

```diff
@@ -314,31 +396,228 @@ static void destroy_cache_entry(CacheEntry* e) {
   free(e);
 }
 
-inline static CacheEntry* get_cache_entry(PyCodeObject* code) {
-  CacheEntry* extra = NULL;
+typedef struct {
+  PyObject_HEAD
+  CacheEntry* cache_entry;
+} CacheEntryWrapper;
+
+static PyTypeObject CacheEntryWrapperType = {
+  PyVarObject_HEAD_INIT(NULL, 0)
+  .tp_name = "torch._C.dynamo.eval_frame.CacheEntryWrapper",
+  .tp_basicsize = sizeof(CacheEntryWrapper),
+  .tp_itemsize = 0,
+  .tp_flags = Py_TPFLAGS_DEFAULT,
+  .tp_new = PyType_GenericNew,
+};
```

Comment:
Are you making this a real Python object because you want to access it from Python?